chalicelib/utils.py

```diff
@@ -44,17 +45,32 @@ def __init__(self, data, url, workspace, namespace, auth=None):
         }
 
     def workspace_exists(self):
-        """Returns true if FireCloud workspace in the specified namespace
-        exists, otherwise false.
-        """
+        """Returns requests response object. Status code is 200 if FireCloud 
```

Comment:
There is a whole range of response codes that can come back, not just 200 and 404.